--- ../orig/rsyslog.conf	2014-08-19 12:59:20.100665000 +0200
+++ rsyslog.conf	2014-09-09 09:20:42.106060439 +0200
@@ -0,0 +1,2 @@
+################################ RSYSLOG-LIB ###################################
+
@@ -5,34 +7,61 @@
 
 #### MODULES ####
 
+##################### RSYSLOG-LIB BEGIN MODULES ################################
+
+##################### RSYSLOG-LIB BEGIN MODLOAD IMUXSOCK #######################
 $ModLoad imuxsock # provides support for local system logging (e.g. via logger command)
+##################### RSYSLOG-LIB END MODLOAD IMUXSOCK #########################
+
+##################### RSYSLOG-LIB BEGIN MODLOAD IMKLOG #########################
 $ModLoad imklog   # provides kernel logging support (previously done by rklogd)
+##################### RSYSLOG-LIB END MODLOAD IMKLOG ###########################
+
+##################### RSYSLOG-LIB BEGIN MODLOAD IMMARK #########################
 #$ModLoad immark  # provides --MARK-- message capability
+##################### RSYSLOG-LIB END MODLOAD IMMARK ###########################
 
+##################### RSYSLOG-LIB BEGIN MODLOAD IMUDP ##########################
 # Provides UDP syslog reception
 #$ModLoad imudp
 #$UDPServerRun 514
+##################### RSYSLOG-LIB END MODLOAD IMUDP ############################
 
+##################### RSYSLOG-LIB BEGIN MODLOAD IMTCP ##########################
 # Provides TCP syslog reception
 #$ModLoad imtcp
 #$InputTCPServerRun 514
+##################### RSYSLOG-LIB END MODLOAD IMTCP ############################
+
+##################### RSYSLOG-LIB END MODULES ##################################
 
 
 #### GLOBAL DIRECTIVES ####
 
-# Use default timestamp format
+##################### RSYSLOG-LIB BEGIN GLOBALS ################################
+
+##################### RSYSLOG-LIB BEGIN MODLOAD OMFILE DEFAULT TEMPLATE ########
+# Use traditional timestamp format
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
+##################### RSYSLOG-LIB END OMFILE DEFAULT TEMPLATE ##################
 
+##################### RSYSLOG-LIB BEGIN MODLOAD OMFILE ENABLE SYNC #############
 # File syncing capability is disabled by default. This feature is usually not required,
 # not useful and an extreme performance hit
 #$ActionFileEnableSync on
+##################### RSYSLOG-LIB END MODLOAD OMFILE ENABLE SYNC ###############
 
+##################### RSYSLOG-LIB BEGIN INCLUDECONFIG ##########################
 # Include all config files in /etc/rsyslog.d/
 $IncludeConfig /etc/rsyslog.d/*.conf
-
+##################### RSYSLOG-LIB END INCLUDECONFIG ############################
+
+##################### RSYSLOG-LIB BEGIN GLOBALS ################################
 
 #### RULES ####
 
+##################### RSYSLOG-LIB BEGIN RULES ##################################
+
 # Log all kernel messages to the console.
 # Logging much else clutters up the screen.
 #kern.*                                                 /dev/console
@@ -78,3 +103,4 @@
 # remote host is: name/ip:port, e.g. 192.168.0.1:514, port optional
 #*.* @@remote-host:514
 # ### end of the forwarding rule ###
+##################### RSYSLOG-LIB END RULES #####################################
